src/main/java/core/basesyntax/stategy/handler/ReturnOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.stategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
```

Comment:
The import statements should be grouped together without empty lines between them.